packages/common/lib/types/Generic.types.ts

```diff
@@ -24,19 +23,24 @@ export interface NameAndLocale {
   locale?: string;
 }
 
+export type NameAndLocaleExtended = NameAndLocale & { [key: string]: unknown };
+
 export interface LogoAndColor {
   logo?: CredentialLogo;
   background_color?: string;
   text_color?: string;
 }
 
-export type Display = NameAndLocale & LogoAndColor & { [key: string]: string };
+export type Display = NameAndLocaleExtended & LogoAndColor;
 
 export interface IssuerMetadata {
   credential_endpoint: string;
   batch_credential_endpoint?: string;
   credentials_supported: CredentialIssuerMetadataSupportedCredentials[];
-  credential_issuer: Display;
+  credential_issuer: string;
+  authorization_server?: string;
```

Comment:
Hmzz, not sure I like that. This means we are getting 2 locations for something which is exactly the same concept. The client and issuer should always have the same payloads so we do not get into situations where the client is behaving differently from the issuer, simply because the interfaces for the payload differ, or we forget to add certain logic in one place.

Reply:
changed the `openid4vci_metadata?: OpenID4VCIServerMetadata` to `openid4vci_metadata?: IssuerMetadata`